client-misk/src/main/kotlin/app/cash/backfila/client/misk/internal/HibernateBackfillOperator.kt

```diff
@@ -342,6 +349,25 @@ internal class HibernateBackfillOperator<E : DbEntity<E>, Pkey : Any, Param : An
       return null
     }
 
+    override fun backfills(): Set<BackfillOperator.BackfillRegistration> {
+      return backfills.map { BackfillOperator.BackfillRegistration(
+          name = it.key,
+          description = (it.value.annotations.find { it is Description } as? Description)?.text,
+          parametersClass = parametersClass(it.value as KClass<Backfill<*, *, Any>>)
+      ) }.toSet()
+    }
+
+    private fun <T : Any> parametersClass(backfillClass: KClass<out Backfill<*, *, T>>): KClass<T> {
```

Comment:
wouldn't this be shared across all types of backfills?

Reply:
No. Dynamo only has two Parameterized types. So this is different enough that it deserves some copy and paste.

I think it is the job of each implementation to specify how to calculate the Parameterized type.

Reply:
Here is my current Dynamo version:
https://github.com/cashapp/backfila/blob/mikepaw.rough-dynamo/client-misk-dynamodb/src/main/kotlin/app/cash/backfila/client/misk/dynamodb/DynamoDbBackfillOperator.kt#L213